Holovibes/sources/gui/windows/panels/export_panel.cc

```diff
@@ -42,6 +36,14 @@ void actualise_record_output_file_ui(const std::filesystem::path file_path)
     NotifierManager::notify<std::filesystem::path>("record_output_file", file_path);
 }
 
+void ExportPanel::init()
+{
+    ui_->NumberOfFramesSpinBox->setSingleStep(record_frame_step_);
+    set_record_mode(static_cast<int>(RecordMode::RAW)); // Not great but it works
+
+    actualise_record_output_file_ui(std::filesystem::path(ui_->OutputFilePathLineEdit->text().toStdString()));
+}
+
 void ExportPanel::on_notify()
 {
     // TODO
```

Comment:
What is this TODO ? If not yours or you don't know it's origin please remove it.